### DISCUSSION TITLE
Explore-Nav missing

### DISCUSSION THREAD

**Discussion by ErSpieltHD** [PoomSmart/YTABConfig#33] [Q&A]
First of all I use uYouPlus.
@qnblackcat said that uYouPlus doesn't have anything to do with the problem. It's rather YouTube's fault.

In the home section the whole Explore menu and sidenav is missing. In the normal YouTube app it's there.

Is there an AB flag to enable?

Explore Menu:
![image](https://user-images.githubusercontent.com/40891294/200367999-ff975d6c-ada2-4b9a-87a9-5c6d31b74cd2.jpeg)

Explore Sidenav:
![image](https://user-images.githubusercontent.com/40891294/200368091-9da311e3-cb09-45fb-bf88-264ef27047c7.jpeg)

**Comment by ErSpieltHD** [comment]
@mrxx71 in #9 has uYouPlus as well and the nav/menu.

https://user-images.githubusercontent.com/117075973/199060945-a757ee2f-6cf2-4f4b-b079-35c86087032f.png

**Comment by arichornlover** [comment]
I tried a lot of flags related to “Navbar” “navigation” and normally “nav” I still couldn’t find the flag you’re looking for and when I tried helping I ended up having to corrupt my YouTube (uYouPlus) again :/
Overall I tried.

**Reply by ErSpieltHD** [reply]
This just frustrates me because the explore tab is gone. I want to keep the shorts tab.

**Reply by arichornlover** [reply]
I think the explore tab was a client-side change, unfortunately that type of change don’t use any flag which is the worst thing that has happened.

**Reply by ErSpieltHD** [reply]
This sucks. I want it to check out trends but have shorts as well
